tools/chkpriv/xrdp-chkpriv.in

```diff
@@ -0,0 +1,199 @@
+#!/bin/sh
+#
+# xrdp: A Remote Desktop Protocol server.
+#
+# Copyright (C) Jay Sorg and contributors 2004-2024
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Program to check permissions for xrdp when running in a non-privileged
+# mode
+
+# Change these if they do not match your installation
+CONF_DIR=/etc/xrdp
```

Comment:
@matt335672 We should use `@sysconfdir@/xrdp` for the default location. It will be set to `/usr/local/etc/xrdp` on FreeBSD.

Reply:
Raised #3159.